tests/topotests/conftest.py

```diff
@@ -66,6 +68,18 @@ def pytest_addoption(parser):
         help="Only set up this topology, don't run tests",
     )
 
+    parser.addoption(
+        "--valgrind-extra",
+        action="store_true",
+        help="Add extra (slower) config to valgrind invocation (if enabled)",
```

Comment:
but what does it do

Reply:
I guess I should split this into the 2 things it does, 1) generate a suppression file (`--valgrind-gen-suppress`?), 2) turn on deeper/more costly inspection, i.e., better checks that cause the process to run slower: `--valgrind-deeper` `--valgrind-better` ? My thinking was that this particular flag would apply to any type of valgrind/helgrind thing turning on extra features that aren't on by default b/c they are to costly/slow.

Reply:
It's fine keeping it as one option, I'm just saying the option help text doesn't say what it does. Imo fix that and it's a merge

Reply:
Ok, latest: https://github.com/FRRouting/frr/pull/8856/files#diff-c06ff0db191a18b40e0d44d66e7d19f63902b02a127323818fb5644023fe5486R80

Look ok?